### ISSUE TITLE
Accessibility Issue:: The heading is missing text

### ISSUE THREAD

**Issue by ddavisgraphics** [wvulibraries/databases_deprecated#90]
Make a decision for this issue
An empty heading occurs.
The tag for the heading is present, but there is no text in the tag.

```
<h2><span class="database-content-title-results"> </span></h2>
```
